packages/devtools_app/lib/src/timeline/timeline_flame_chart.dart

```diff
@@ -222,46 +222,40 @@ class TimelineFlameChartState
       _timelineController.selectedFrame,
       _handleSelectedFrame,
     );
-
-    addAutoDisposeListener(_timelineController.activeSearchMatch, () async {
-      final activeSearch = _timelineController.activeSearchMatch.value;
-      if (activeSearch == null) return;
-
-      // Ensure the [activeSearch] is vertically in view.
-      if (!_isEventVerticallyInView(activeSearch)) {
-        await scrollVerticallyToEvent(activeSearch);
-      }
-
-      // TODO(kenz): zoom if the event is less than some min width.
-
-      // Ensure the [activeSearch] is horizontally in view.
-      if (!_isEventHorizontallyInView(activeSearch)) {
-        await scrollHorizontallyToTime(
-            timeMicros: activeSearch.time.start.inMicroseconds);
-      }
-    });
   }
 
-  bool _isEventVerticallyInView(TimelineEvent event) {
-    final eventTopY = _topYForEvent(event);
+  @override
+  bool isDataVerticallyInView(TimelineEvent data) {
+    final eventTopY = topYForData(data);
     return eventTopY > verticalScrollOffset &&
         eventTopY + rowHeightWithPadding <
             verticalScrollOffset + widget.containerHeight;
   }
 
-  bool _isEventHorizontallyInView(TimelineEvent event) {
-    return visibleTimeRange.contains(event.time.start) &&
-        visibleTimeRange.contains(event.time.end);
+  @override
+  bool isDataHorizontallyInView(TimelineEvent data) {
+    return visibleTimeRange.contains(data.time.start) &&
```

Comment:
Not new code but I think there is a bug.
This will return false if `data` is a range that starts before the visibleTimeRange and end after the visibleTimeRange. Elsewhere you have the proper range intersection calculation of checking the a.start <= b.end && a.end >= a.start

Reply:
fixed